content/knowledge-base/firebase-crashlytics-dsym-uploading.md

```diff
@@ -41,6 +41,22 @@ In order to upload the dSYM files to Firebase Crashlytics, add the following scr
     $CM_BUILD_DIR/ios/Pods/FirebaseCrashlytics/upload-symbols -gsp ios/Runner/GoogleService-Info.plist -p ios $dsymPath
   fi
   ```
+ 
+The above-mentioned **dsymPath** is Flutter specific and it could change depending on what platform the app is built. 
+
+With React Native applications:
+
+```
+dsymPath=$(find $CM_BUILD_DIR/build/ios/xcarchive/codemagic_react_native_*.xcarchive -name "*.dSYM" | head -1)
+```
+
+With Native iOS applications:
+
+```
+dsymPath=$(find $CM_BUILD_DIR/build/ios/xcarchive/swiftly_*.xcarchive -name "*.dSYM" | head -1)
+```
```

Comment:
The example paths have the names of the apps you were testing with. Is it possible to use the wildcard for the whole string? For example  `/*.xcarchive`?

Also, is there actually any difference between the path for Native iOS and React Native? If not, just show one path and say this can be used for both application types.